Makefile

```diff
@@ -57,10 +57,7 @@ build-ra-language-french:
 
 build-react-admin:
 	@echo "Transpiling react-admin files...";
-	@rm -rf ./packages/react-admin/docs
```

Comment:
You should probably keep this one for those of us who have an old copy of the docs